sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/VectorIndexType.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Defines the index type of vector index specification in the Azure Cosmos DB service.
+ */
+public enum VectorIndexType {
```

Comment:
`CosmosVectorIndexType` following the `CosmosVectorDataType` ?

Reply:
Updated the code